temporal-test-server/src/main/java/io/temporal/serviceclient/TestServiceStubsOptions.java

```diff
@@ -0,0 +1,75 @@
+/*
+ *  Copyright (C) 2020 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.serviceclient;
+
+public final class TestServiceStubsOptions extends ServiceStubsOptions {
+  private static final TestServiceStubsOptions DEFAULT_INSTANCE =
+      newBuilder().validateAndBuildWithDefaults();
+
+  public static Builder newBuilder() {
+    return new Builder();
+  }
+
+  public static Builder newBuilder(TestServiceStubsOptions options) {
+    return new Builder(options);
+  }
+
+  public static Builder newBuilder(ServiceStubsOptions options) {
+    return options instanceof TestServiceStubsOptions
+        ? newBuilder((TestServiceStubsOptions) options)
+        : new Builder(options);
```

Comment:
Well, this structure was copied from `WorkflowServiceStubsOptions` and it makes sense there. Here is doesn't, but I don't want to remove it. The only reason it doesn't make sense right now is because the TestServiceStubsOptions don't add anything to ServiceStubsOptions.

Reply:
On the second thought... let me restructure the code to both leave the framework in place and don't leave such a strange-looking code.